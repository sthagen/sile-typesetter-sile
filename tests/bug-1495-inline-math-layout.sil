\begin[papersize=a6]{document}
\use[module=packages.retrograde,target=v0.15.0]
\nofolios
\neverindent
\use[module=packages.math]
Lorem ipsum dolor sit amet, consectetur \math{1 + 1 = 2} elit.
Morbi laoreet diam odio, faucibus gravida massa \math{e^{i\pi} = \mi{cos}(\pi) = -1} vitae.
Maecenas venenatis ultrices.
Phasellus \math{e^{i\theta} = \mi{cos}(\theta) + i\,\mi{sin}(\theta)} elementum quam, eget lobortis ligula bibendum ac.
Etiam cursus vulputate nisl at convallis.
Aliquam ipsum purus, maximus a felis quis, vehicula efficitur tortor.
\end{document}
