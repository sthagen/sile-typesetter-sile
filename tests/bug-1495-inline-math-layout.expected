Set paper size 	297.6377985	419.5275636
Begin page
Mx 	14.8819
My 	28.5447
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	47 82 85 72 80 w=26.4014	(Lorem)
Mx 	44.8661
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	73.6492
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	99.1607
T	86 76 87 w=10.0146	(sit)
Mx 	112.7582
T	68 80 72 87 w=20.6836	(amet)
Mx 	133.4418
T	15 w=2.2900	(,)
Mx 	139.3147
T	70 82 81 86 72 70 87 72 87 88 85 w=48.5303	(consectetur)
Mx 	191.4279
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	18 w=4.6500	(nil)
Mx 	199.2489
T	12 w=6.4800	(nil)
Mx 	208.9000
T	18 w=4.6500	(nil)
Mx 	218.6999
T	30 w=6.4800	(nil)
Mx 	230.3298
T	19 w=4.6500	(nil)
Mx 	238.5627
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	72 79 76 87 w=13.4814	(elit)
Mx 	252.0441
T	17 w=2.2900	(.)
Mx 	257.9170
T	48 82 85 69 76 w=24.8389	(Morbi)
Mx 	14.8819
My 	40.5447
T	79 68 82 85 72 72 87 w=28.9697	(laoreet)
Mx 	46.8857
T	71 76 68 80 w=20.5322	(diam)
Mx 	70.4520
T	82 71 76 82 w=17.9688	(odio)
Mx 	88.4207
T	15 w=2.2900	(,)
Mx 	93.7448
T	73 68 88 70 76 69 88 86 w=34.3164	(faucibus)
Mx 	131.0953
T	74 85 68 89 76 71 68 w=30.8594	(gravida)
Mx 	164.9888
T	80 68 86 86 68 w=24.9365	(massa)
Mx 	192.9594
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	196.7694
My 	37.4447
Set font 	Libertinus Math;8;400;Regular;normal;;;LTR
T	2753 w=1.8320	(nil)
Mx 	198.6014
T	874 w=4.5280	(nil)
Mx 	207.3248
My 	40.5447
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	217.5702
T	68 80 84 w=13.2200	(nil)
Mx 	230.7902
Mx 	2.8200
T	9 a=3.5600	(nil)
Mx 	234.3502
T	874 w=5.6600	(nil)
Mx 	240.0102
Mx 	2.8200
T	10 a=3.5600	(nil)
Mx 	247.3357
T	30 w=6.4800	(nil)
Mx 	257.5811
T	2088 w=6.4800	(nil)
Mx 	264.0611
T	18 w=4.6500	(nil)
Mx 	271.7452
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	89 76 w=7.6416	(vi)
Mx 	279.3868
T	16 w=3.3691	(-)
Mx 	14.8819
My 	52.6701
T	87 68 72 w=12.6514	(tae)
Mx 	27.5333
T	17 w=2.2900	(.)
Mx 	32.3980
T	48 68 72 70 72 81 68 86 w=40.2295	(Maecenas)
Mx 	75.2022
T	89 72 81 72 81 68 87 76 86 w=39.8096	(venenatis)
Mx 	117.5865
T	88 79 87 85 76 70 72 86 w=30.9717	(ultrices)
Mx 	148.5582
T	17 w=2.2900	(.)
Mx 	153.4230
T	51 75 68 86 72 79 79 88 86 w=38.3984	(Phasellus)
Mx 	194.3961
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	2750 w=3.8100	(nil)
Mx 	198.2061
My 	49.5701
Set font 	Libertinus Math;8;400;Regular;normal;;;LTR
T	2753 w=1.8320	(nil)
Mx 	200.0381
T	866 w=3.9120	(nil)
Mx 	207.1579
My 	52.6701
Set font 	Libertinus Math;10;400;Regular;normal;;;LTR
T	30 w=6.4800	(nil)
Mx 	216.4157
T	68 80 84 w=13.2200	(nil)
Mx 	229.6357
Mx 	3.0000
T	3798 a=4.2100	(nil)
Mx 	233.8457
T	866 w=4.8900	(nil)
Mx 	238.7357
Mx 	2.9900
T	3799 a=4.2100	(nil)
Mx 	244.9625
T	12 w=6.4800	(nil)
Mx 	253.4592
T	2753 w=2.2900	(nil)
Mx 	257.4159
T	84 74 79 w=12.0300	(nil)
Mx 	269.4459
Mx 	3.0000
T	3798 a=4.2100	(nil)
Mx 	273.6559
T	866 w=4.8900	(nil)
Mx 	278.5459
Mx 	2.9900
T	3799 a=4.2100	(nil)
Mx 	14.8819
My 	64.6701
Set font 	Gentium Plus;10;400;;normal;;;LTR
T	72 79 72 80 72 81 87 88 80 w=46.8896	(elementum)
Mx 	64.2939
T	84 88 68 80 w=23.0713	(quam)
Mx 	87.3652
T	15 w=2.2900	(,)
Mx 	92.1775
T	72 74 72 87 w=17.5586	(eget)
Mx 	112.2585
T	79 82 69 82 85 87 76 86 w=31.8213	(lobortis)
Mx 	146.6021
T	79 76 74 88 79 68 w=22.8955	(ligula)
Mx 	172.0199
T	69 76 69 72 81 71 88 80 w=41.5332	(bibendum)
Mx 	216.0755
T	68 70 w=8.9600	(ac)
Mx 	225.0355
T	17 w=2.2900	(.)
Mx 	229.8478
T	40 87 76 68 80 w=23.7354	(Etiam)
Mx 	256.1055
T	70 88 85 86 88 86 w=26.6504	(cursus)
Mx 	14.8819
My 	76.6701
T	89 88 79 83 88 87 68 87 72 w=39.6289	(vulputate)
Mx 	56.9428
T	81 76 86 79 w=14.7998	(nisl)
Mx 	74.1745
T	68 87 w=8.0322	(at)
Mx 	84.6387
T	70 82 81 89 68 79 79 76 86 w=36.4307	(convallis)
Mx 	121.0694
T	17 w=2.2900	(.)
Mx 	125.7914
T	36 79 76 84 88 68 80 w=34.4434	(Aliquam)
Mx 	162.6667
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	190.2989
T	83 88 85 88 86 w=23.7158	(purus)
Mx 	214.0147
T	15 w=2.2900	(,)
Mx 	218.7367
T	80 68 91 76 80 88 86 w=37.5537	(maximus)
Mx 	258.7224
T	68 w=4.5898	(a)
Mx 	265.7442
T	73 72 79 76 86 w=17.0117	(felis)
Mx 	14.8819
My 	88.6701
T	84 88 76 86 w=17.0215	(quis)
Mx 	31.9034
T	15 w=2.2900	(,)
Mx 	36.8520
T	89 72 75 76 70 88 79 68 w=34.7461	(vehicula)
Mx 	74.2567
T	72 741 70 76 87 88 85 w=33.3301	(efficitur)
Mx 	110.2453
T	87 82 85 87 82 85 w=24.8633	(tortor)
Mx 	135.1086
T	17 w=2.2900	(.)
End page
Finish
